internal/compiler/generator/cpp.rs

```diff
@@ -2776,6 +2776,21 @@ fn compile_expression(expr: &llr::Expression, ctx: &EvaluationContext) -> String
                 )
 
         }
+        Expression::MinMax { ty, op, lhs, rhs } => {
+            let ident = match op {
+                MinMaxOp::Min => "min",
+                MinMaxOp::Max => "max",
+            };
+            let lhs_code = return_compile_expression(lhs, ctx, Some(&ty));
+            let rhs_code = return_compile_expression(rhs, ctx, Some(&ty));
```

Comment:
```suggestion
            let lhs_code = compile_expression(lhs, ctx);
            let rhs_code = compile_expression(rhs, ctx);
```

This is the cause of the CI failure :-)

Reply:
Done, thanks